copyright

```diff
@@ -5,6 +5,7 @@ Source: https://github.com/endless-sky/endless-sky
 
 Files: *
 Copyright: Michael Zahniser <mzahniser@gmail.com>
+           endless-sky contributors (see credits.txt and changelog)
```

Comment:
This should probably be a comment like with other entries.
```
Files:
 images/outfit/*battery?hai*
 ...
Copyright: Maximilian Korber
License: CC-BY-SA-4.0
Comment: Derived from works by Michael Zahniser (under the same license).
```
Perhaps also mention the copyright headers in the text and source files.

Reply:
Do you recommend removing "endless-sky contributors"?

```
Files: *
Copyright: Michael Zahniser <mzahniser@gmail.com>
           endless-sky contributors
Comment: For more details, see credits.txt, changelog, and the copyright headers in the text and source files.
```
or
```
Files: *
Copyright: Michael Zahniser <mzahniser@gmail.com>
Comment: For more details, see credits.txt, changelog, and the copyright headers in the text and source files.
```

Reply:
This is an appropriate format. It should not be a comment. Ref [the copyright format spec](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/):

> Formatted text, no synopsis: one or more free-form copyright statements. Any formatting is permitted; 

Ref "Formatted text":

> Formatted text fields use the same rules as the long description in a package's `Description` field in Debian control files. 

[Package `Description`](https://www.debian.org/doc/debian-policy/ch-controlfields#description):

> The lines in the extended description can have these formats:
-     Those starting with a single space are part of a paragraph. Successive lines of this form will be word-wrapped when displayed. The leading space will usually be stripped off. The line must contain at least one non-whitespace character.
-     Those starting with two or more spaces. These will be displayed verbatim. If the display cannot be panned horizontally, the displaying program will line wrap them “hard” (i.e., without taking account of word breaks). If it can they will be allowed to trail off to the right. None, one or two initial spaces may be deleted, but the number of spaces deleted from each line will be the same (so that you can have indenting work correctly, for example). The line must contain at least one non-whitespace character.
-     Those containing a single space followed by a single full stop character. These are rendered as blank lines. This is the only way to get a blank line.
-     Those containing a space, a full stop and some more characters. These are for future expansion. Do not use them.

> Do not use tab characters. Their effect is not predictable.


Also the last example in the copyright spec shows a similar use:
```
Files: *
Copyright: 2008, John Doe <jdoe@example.com>
           2007, Jane Smith <jsmith@example.org>
           2007, Joe Average <joe@example.org>
           2007, J. Random User <jr@users.example.com>
License: PSF-2
```